app-modules/consultants/src/Policies/DocumentPolicy.php

```
@@ -0,0 +1,42 @@
+
+namespace TresPontosTech\Consultants\Policies;
+
+use App\Models\Users\User;
+use Illuminate\Auth\Access\HandlesAuthorization;
+use TresPontosTech\Consultants\Models\Document;
+
+class DocumentPolicy
+{
+    use HandlesAuthorization;
+
+    public function viewAny(User $user): bool
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$user'. (undefined)

(UnusedFormalParameter)